core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -53,6 +53,7 @@
                     !isOnTerminalCard()">
           <md-button aria-label="" class="oppia-learner-continue-button protractor-test-continue-to-next-card-button" focus-on="<[::CONTINUE_BUTTON_FOCUS_LABEL]>" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" translate="I18N_PLAYER_CONTINUE_BUTTON">
           </md-button>
+          <img class="logo-bottom-nav-bar" ng-click="showPendingCard(upcomingStateName, upcomingParams, upcomingContentHtml)" src="/assets/images/icons/626x626_continue_icon.png">
```

Comment:
I recommend taking a look at other icons in Oppia -- see if you can reuse stuff from e.g. material design, and search for the class "material-icons". Try to follow that pattern.